libfwupdplugin/fu-ble-device.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2021 Ricardo Cañuelo <ricardo.canuelo@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#define G_LOG_DOMAIN				"FuBleDevice"
+
+#include "config.h"
+
+#include "fu-device-private.h"
+#include "fu-ble-device.h"
+
+
+/**
+ * SECTION:fu-ble-device
+ * @short_description: a Bluetooth LE device
+ *
+ * An object that represents a Bluetooth LE device.
+ *
+ * See also: #FuDevice
+ */
+
+typedef struct {
+	gchar		*name;
+	gchar		*address;
+	gchar		*adapter;
```

Comment:
I don't actually see adapter used.  Is that an oversight/mistake or not yet implemented?

Reply:
yet implemented, and I added it